packages/app/src/cli/models/extensions/specifications/app_config_app_access.ts

```diff
@@ -5,7 +5,9 @@ import {zod} from '@shopify/cli-kit/node/schema'
 const AppAccessSchema = zod.object({
   access: zod
     .object({
-      direct_api_offline_access: zod.boolean().optional(),
+      admin: zod
+        .object({direct_api_mode: zod.union([zod.literal('disabled'), zod.literal('online'), zod.literal('offline')])})
+        .optional(),
     })
     .optional(),
```

Comment:
We went through a Config API Alignment process, and originally settled on an API that looked like checkout's  `api_access` [config](https://shopify.dev/docs/api/checkout-ui-extensions/unstable/configuration#how-it-works) (Checkout: `[extensions.capabilities.api_access]`, Option 5 from RFC: `[capabilities.api_access.admin]`). But after some concerns about `[capabilities]` being too vague at the app level and `[access]` being more explicit, I decided to move forward with `[access]`. 

More info: https://github.com/Shopify/ui-api-design/discussions/191#discussioncomment-8112987

Reply:
Happy with Option 6. Overall very similar to the agreed upon Option 5, but is more accurate and specific at the app root level. Overall I don't think it's too much of a departure from the Option 5 proposal for this to be a big difference. It still follows the root of `access` -> `admin` and is now explicit about the `direct_api` feature.